lib/dockerize_rails/docker_commands/docker_start.rb

```diff
@@ -0,0 +1,77 @@
+module DockerizeRails
+  module DockerCommands
+    module DockerStart
+      def self.start_rails
+        services = DockerizeRails::DockerCommands::Helpers.get_services_from_docker_compose
+        docker_start(services['rails'], "#{DRConfig.application_name}_rails:#{DRConfig.application_env}") \
+        if services.has_key? 'rails'
```

Comment:
__[rubocop]__
* Use `Hash#key?` instead of `Hash#has_key?`. [:ref](https://github.com/bbatsov/ruby-style-guide#hash-key)

<!-- Comment by [SideCI](https://sideci.com) -->